src/mame/apple/apple2video.h

```diff
@@ -73,11 +73,8 @@ class a2_video_device : public device_t, public device_palette_interface, public
 	void init_palette();
 
 private:
-	template <bool iie, bool invert, bool flip>
+	template<Model model, bool invert, bool flip>
 	void plot_text_character(bitmap_ind16 &bitmap, int xpos, int ypos, int xscale, uint32_t code, int fg, int bg);
```

Comment:
The space between `template` and the opening angle bracket in the declaration is intentional.  Also, recommendation in MAME is to use `LlamaCase` for template parameter names as they’re semantically different from variables.  We also use `snake_case` for type names and `SCREAMING_SNAKE_CASE` for constants.

See here: https://docs.mamedev.org/contributing/cxx.html#naming-conventions